src/app/tests/TestCommandInteraction.cpp

```diff
@@ -91,6 +92,30 @@ namespace app {
 
 CommandHandler::Handle asyncCommandHandle;
 
+struct ForcedSizeBuffer
+{
+    chip::Platform::ScopedMemoryBufferWithSize<uint8_t> mBuffer;
+
+    ForcedSizeBuffer(uint32_t size)
+    {
+        if (mBuffer.Alloc(size))
+        {
+            memset(mBuffer.Get(), 0x12, size);
```

Comment:
Why 0x12?

Reply:
Just some value to use. Added comment to mention that it doesn't have any significance

Reply:
Fair random number generated by rolling a D20 .... :)